jsk_fetch_robot/jsk_fetch_startup/launch/fetch_bringup.launch

```diff
@@ -11,42 +17,54 @@
     <param name="buffer_size" value="120.0"/>
   </node>
 
-  <!-- jsk_pr2_startup/jsk_pr2_lifelog/db_client.launch -->
   <!-- mongodb -->
-  <param name="robot/name" value="fetch" />
   <include file="$(find jsk_robot_startup)/lifelog/mongodb.launch" >
     <arg name="use_daemon" value="true"/>
     <arg name="port" value="27017" />
     <arg name="repl_set_mode" value="false" />
     <arg name="replicate" default="false" />
   </include>
-  <!-- logging and resoring pr2 position data -->
-  <arg name="map_frame" value="map" />
+
+  <!-- logging and resoring robot position data -->
   <node name="move_base_db"
         pkg="jsk_pr2_startup" type="move_base_db.py" >
     <param name="map_frame" value="$(arg map_frame)"/>
     <param name="robot_frame" value="base_link"/> <!-- fetch does not have base_footprint -->
     <param name="update_cycle" value="0.1" />
   </node>
 
-  <!-- include fetch_navigation -->
-  <include file="$(find fetch_navigation)/launch/fetch_nav.launch" >
-    <arg name="map_file" default="$(find jsk_maps)/raw_maps/eng2-7f-0.05.yaml" />
-    <arg name="map_keepout_file" default="$(find jsk_maps)/raw_maps/eng2-7f-0.05_keepout.yaml" />
-    <arg name="use_keepout" default="true" />
-  </include>
-  <rosparam ns="move_base/global_costmap">
+  <!-- jsk maps -->
+  <include file="$(find jsk_maps)/launch/start_map_$(arg map_frame).launch" />
+
+  <!-- include fetch moveit -->
+  <include file="$(find fetch_moveit_config)/launch/move_group.launch"
+           if="$(arg launch_moveit)" />
+
+  <group if="$(arg launch_move_base)">
```

Comment:
ok, i understand, keepout feature is very important for real robot, so I rewrite this PR without multimap floor map system
https://github.com/jsk-ros-pkg/jsk_robot/pull/692